qiita_db/support_files/patches/89.sql

```diff
@@ -0,0 +1,8 @@
+-- Nov 1, 2023
+-- add creation_job_id to qiita.prep_template
+ALTER TABLE qiita.analysis ADD slurm_reservation VARCHAR DEFAULT '' NOT NULL;
+ALTER TABLE qiita.user_level ADD slurm_parameters VARCHAR DEFAULT '--nice 10000' NOT NULL;
+
+UPDATE qiita.user_level SET slurm_parameters = '--nice 5000' WHERE name = 'admin';
+
+UPDATE qiita.user_level SET slurm_parameters = '' WHERE name = 'wet-lab admin';
```

Comment:
wet-lab admins are between regular users and admins, but it seems here that wet-lab admins will have the same priority as normal users. This is okay if slurm_parameters is only used for things like SPP jobs, but in general wet-lab admins should have higher priority than normal users right?

Reply:
The other way around, all users have a --nice 10000 value, which wet-lab admin have no value - this means that they will have the highest priority. In other words, the higher nice value the lower priority it has.